Docs/Configuration.tex

```diff
@@ -112,31 +112,31 @@ \section{Introduction}\label{introduction}
 
 This document provides information on
 \href{https://github.com/acidanthera/OpenCorePkg}{OpenCore} user
-configuration file format used to setup the correct functioning of macOS
+configuration file format used to setup the correct functioning of the macOS
 operating system. It is to be read as the official clarification of expected
 OpenCore behaviour. All deviations, if found in published OpenCore releases,
-shall be considered documentation or implementation bugs, and are requested to be
-reported through \href{https://github.com/acidanthera/bugtracker}{Acidanthera Bugtracker}.
-Errata sheet is available in
+shall be considered to be documentation or implementation bugs which should be
+reported via the \href{https://github.com/acidanthera/bugtracker}{Acidanthera Bugtracker}.
+An errata sheet is available in
 \href{https://github.com/acidanthera/OpenCorePkg/blob/master/Docs/Errata/Errata.pdf}{OpenCorePkg repository}.
 
-This document is structured as a specification, and is not meant to provide a step by
-step algorithm for configuring end-user board support package (BSP). The intended audience
-of the document are programmers and engineers with basic understanding of macOS internals
-and UEFI functioning. For these reasons this document is available exclusively in English,
+This document is structured as a specification and is not meant to provide a step by
+step guide to configuring an end-user Board Support Package (BSP). The intended audience
+of the document are programmers and engineers with a basic understanding of macOS internals
```

Comment:
...of the documents *is*....

(The subject of this sentence is audience, which has single form)

Reply:
I don't think you can use "is" for "programmers" and "engineers"

Reply:
Hmm, how about *The target audience is those with a basic understanding of*...

Reply:
The shorter the better, the `is anticipated` way might actually be more idiomatic.